bindings/python/tools/python/ostk/astrodynamics/display.py

```
@@ -210,9 +232,60 @@ def add_satellite(
+    ) -> None:
+        """
+        Add a satellite trajectory to the plot, including a highligh of the accesses.
+
+        Args:
+            trajectory (Trajectory): The satellite trajectory.
+            accesses (list[Access]): The list of accesses.
+            rgb (list[int]): The RGB color of the satellite.
+            opacity (float): The opacity of the accesses.
+        """
+        if accesses:
+            warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)